morecantile/models.py

```diff
@@ -1327,16 +1329,27 @@ def feature(
             otherwise original coordinate values will be preserved (default).
         projected : bool, optional
             Return coordinates in TMS projection. Default is false.
+        geographic_crs: pyproj.CRS, optional
+            Geographic CRS to use when `projected=False`. Default to 'EPSG:4326' as per GeoJSON specification.
+            .
 
         Returns
         -------
         dict
 
         """
+        geographic_crs = geographic_crs or WGS84_CRS
```

Comment:
> Below in the transform call doing or WGS84_CRS here will cause issues for non-earth CRS (proj will raise an error for the celestial body).

Yeah, but basically I was assuming that GeoJSON should be WGS84 by default. But this can be overridden by passing a CRS (any geographic CRS users want).

Reply:
> doesn't the TileMatrixSet model still grab the geodetic_crs from the crs of the TMS?

@AndrewAnnex I realized that I didn't really answered your question last time. Yes it should take the geographic CRS from the TMS's CRS but it's not guaranteed that it will be EPSG:4326 (as mentioned in the spec). If we look at the original issue #164, we broke `.features` method for most of the TMS which I don't think is a good thing because >50% of the morecantile users will work on earth 😅 

That's said, with the proposed PR, we keep the ability to produce geojson with non-epsg4326 CRS, the user just needs to pass the `geographic_crs`

Reply:
heh there's things a lot cooler than Earth to look at ;)

I broadly think this is fine so long as there is an obvious/easy way to provide a correct CRS even if it's not 4326 and not used directly from the TMS, and so long as code using this method maybe in titiler or such are aware of that... to avoid additional PRs.

Reply:
titiler doesn't use `tms.feature` method 👍